Task Queues and Workers (#708)
temporalio/documentation: docs/content/what-is-a-sticky-execution.md

---
id: what-is-a-sticky-execution
title: What is a Sticky Execution?
description: A Sticky Execution is a when a Worker Entity caches the Workflow Execution Event History and creates a dedicated Task Queue to listen on.
tags:
  - explanation
---

A Sticky Execution is a when a Worker Entity caches the Workflow Execution Event History and creates a dedicated Task Queue to listen on.

A Sticky Execution occurs after a Worker Entity completes the first Workflow Task in the chain of Workflow Tasks for the Workflow Execution.

> The Worker Entity will have cached the Workflow Execution Event History and begin polling the Sticky Queue for Workflow Tasks that contain incremental updates, rather than the entire Event History.

Author: ```suggestion
The Worker Entity caches the Workflow Execution Event History and begins polling the dedicated Task Queue for Workflow Tasks that contain updates, rather than the entire Event History.
```
